cc/toolchains/llvm/cc_toolchain_config.bzl

```
@@ -35,7 +35,7 @@ def cc_toolchain_config(
     if not is_target_triplet(target_system_name):
         fail(target_system_name + " is not a target tripplet")
 
-    cross_compile = host_system_name != target_system_name
+    use_bundled_libcpp = builtin_sysroot != None and not is_darwin
```

```suggestion
    use_libstdcpp = builtin_sysroot != None and not is_darwin
```

nit - the variable name doesn't match the logic. When this is true we should use `libstdc++` not `libc++`